packages/round-manager/src/features/api/services/program.ts

```diff
@@ -0,0 +1,91 @@
+import { ethers } from "ethers"
+import { api } from ".."
+import { global } from "../../../global"
+import { programFactoryContract } from "../contracts"
+import { Program } from "../types"
+
+
+export const programApi = api.injectEndpoints({
+  endpoints: (builder) => ({
+    createProgram: builder.mutation<string, Program>({
+      queryFn: async ({ metadataIdentifier, operatorWallets }) => {
+        try {
+          const programFactory = new ethers.Contract(
+            programFactoryContract.address,
+            programFactoryContract.abi,
+            global.web3Signer
+          )
+
+          // Deploys a new Program contract
+          // Learn about the metadata pointer here: 
+          // https://github.com/gitcoinco/grants-round/blob/main/packages/contracts/docs/MetaPtrProtocol.md
+          let tx = await programFactory.create(
+            { protocol: 1, pointer: metadataIdentifier },
```

Comment:
as opposed to hardcoding it as 1 here. Let's define it as a const object 

```
const MetaPtrProtocol = {
  "ipfs": 1
}
```

This could live in it's own file (assuminging we use it across the repo)

Reply:
changed in #41 

will be implemented in follow-up pr